### PR TITLE
Python 3: decode system_output result to text

### PR DESCRIPTION
**Pull request by hsmj1412**
Signed-off-by: Junxiang Li <junli@redhat.com>

### CODE REVIEW

File: virttest/libvirt_vm.py [hunk 1]
```diff
@@ -19,19 +19,20 @@
...
+from virttest import remote
+from virttest import virsh
+from virttest import libvirt_xml
+from virttest import data_dir
+from virttest import xml_utils
+from virttest import utils_selinux
+from virttest import test_setup
+from virttest import utils_package
+from virttest.compat_52lts import results_stdout_52lts, results_stderr_52lts
+from virttest.compat_52lts import decode_to_text
```
Comment (chunfuwen) [inline virttest/libvirt_vm.py]: merged line 34 and 35

File: virttest/qemu_storage.py [hunk 2]
```diff
@@ -14,12 +14,13 @@
...
-from . import storage
-from . import data_dir
-from . import error_context
-from .compat_52lts import results_stdout_52lts, results_stderr_52lts
+from virttest import utils_misc
+from virttest import virt_vm
+from virttest import storage
+from virttest import data_dir
+from virttest import error_context
+from virttest.compat_52lts import results_stdout_52lts, results_stderr_52lts
```
Comment (chunfuwen) [inline virttest/qemu_storage.py]: merged line 22 and 23

File: virttest/staging/lv_utils.py [hunk 3]
```diff
@@ -47,8 +47,9 @@
 from avocado.core import exceptions
 from avocado.utils import process
 
-from .. import error_context
-from ..compat_52lts import results_stdout_52lts, results_stderr_52lts
+from virttest import error_context
+from virttest.compat_52lts import results_stdout_52lts, results_stderr_52lts
```
Comment (chunfuwen) [inline virttest/staging/lv_utils.py]: merged line 50 and 51

File: virttest/utils_misc.py [hunk 4]
```diff
@@ -58,6 +58,7 @@
 from virttest.staging import utils_koji
 from virttest.xml_utils import XMLTreeFile
 from virttest.compat_52lts import results_stdout_52lts, results_stderr_52lts
+from virttest.compat_52lts import decode_to_text
```
Comment (chunfuwen) [inline virttest/utils_misc.py]: merged to line 60

File: virttest/utils_net.py [hunk 5]
```diff
@@ -27,6 +27,7 @@
 from virttest import utils_misc
 from virttest import arch
 from virttest.compat_52lts import results_stdout_52lts, results_stderr_52lts
+from virttest.compat_52lts import decode_to_text
```
Comment (chunfuwen) [inline virttest/utils_net.py]: merged to line 29

File: virttest/utils_test/libvirt.py [hunk 6]
```diff
@@ -36,37 +36,37 @@
...
+from virttest import libvirt_storage
+from virttest import utils_net
+from virttest import gluster
+from virttest import remote
+from virttest import test_setup
+from virttest import data_dir
+from virttest.utils_iptables import Iptables
+from virttest.staging import lv_utils
+from virttest.utils_libvirtd import service_libvirtd_control
+from virttest.compat_52lts import results_stdout_52lts, results_stderr_52lts
```
Comment (chunfuwen) [inline virttest/utils_test/libvirt.py]: merged line 54 and 55

**Comment by hsmj1412** [comment]
@rdkdd the str default encoding is ascii on Python2, the error means it is a other encoding string inputed. And decode_to_text use sys.getdefaultencoding() to be consistent with avocado's CmdResult.stdout_text. I'm not sure how to fix it is better.  
@ldoktor Any update about https://github.com/avocado-framework/avocado/pull/2569 ?

**Comment by ldoktor** [comment]
Hello @hsmj1412, check-out the https://www.redhat.com/archives/avocado-devel/2018-April/msg00002.html RFC which talks about this. It's still in progress but for now I'd suggest hardcoding "utf-8" as other places do this kind of magic and later we'll address these systematically.

**Comment by ldoktor** [comment]
PS: Do get involved in the RFC.

**Pull request merged**